integration/spark/src/main/scala/org/apache/carbondata/datamap/IndexRebuildRDD.scala

```diff
@@ -101,9 +103,11 @@ object IndexRebuildRDD {
       sparkSession,
       new RefreshResultImpl(),
       carbonTable.getTableInfo,
+      schema,
       schema.getDataMapName,
       indexedCarbonColumns.asScala.toArray,
-      segments2DmStorePath.keySet
+      segments2DmStorePath.keySet,
+      schema.getProviderName
```

Comment:
just passing schema is enough, 
schema.getProviderName, schema.getDataMapName, can be obtained inside from this